Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: help lint lint-fix
+help:
+	@echo "  lint                   Check style with flake8, black and isort"
+	@echo "  lint                   Fix style with flake8, black and isort"
+	@echo "  lint-fix               Format code with black and isort"
+
+lint: build
+	docker-compose run app flake8
+	docker-compose run app python -m black --check .
+	docker-compose run app python -m isort --check-only .
+
+lint-fix: build
+	docker-compose run app python -m black .
+	docker-compose run app python -m isort .
```

Comment:
This clearly won't work, as we're not using `docker-compose`. 😄 

Two things:

1. Test your code by running `make lint` and `make lint-fix`
2. Have the .circleci config above call the makefile.

Reply:
Oops! 
Okay, so I have made some changes and tested them as well. 
Also, I have added another linting library called autoflake, which would remove unused imports. Before I had added this, they were just being flagged by flake8 on running `make lint`. Now, on running `make lint-fix`, that would get fixed as well. 
Let me know your thoughts on this one.